superset-frontend/src/explore/controlPanels/Area.js

```diff
@@ -48,34 +62,30 @@ export default {
           },
         ],
         ['color_scheme', 'label_colors'],
-        ['rich_tooltip', 'show_controls'],
+        [richTooltip, showControls],
       ],
     },
     {
       label: t('X Axis'),
       expanded: true,
       controlSetRows: [
-        ['x_axis_label', 'bottom_margin'],
-        ['x_ticks_layout', 'x_axis_format'],
-        ['x_axis_showminmax', null],
+        [xAxisLabel, bottomMargin],
+        [xTicksLayout, xAxisFormat],
```

Comment:
Was the `controlOverrides` applied to `xAxisFormat`?

Reply:
Effectively, yes. The majority of the `xAxisFormat` controls had the exact same overrides, so I made _that_ the default configuration whem moving it to `Shared_NVD3.js`, and instead added overrides the _other_ way. I figured the majority use case should be the default :)